src/platforms/android/configuration/integrations/file-io.mdx

```diff
@@ -0,0 +1,139 @@
+---
+title: File I/O
+caseStyle: camelCase
+supportLevel: production
+categories:
+  - mobile
+---
+
+<Note>
+
+Supported in Sentry's Android SDK version `5.5.0` and above.
+
+Supported in Sentry Android Gradle Plugin version `3.0.0` and above.
+
+</Note>
+
+The [Sentry Android Gradle Plugin](/platforms/android/gradle/) provides File I/O support through bytecode manipulation. The source can be found [on GitHub](https://github.com/getsentry/sentry-android-gradle-plugin/tree/main/plugin-build/src/main/kotlin/io/sentry/android/gradle/instrumentation).
+
+On this page, we get you up and running with Sentry's File I/O Integration, so that it will automatically start a span out of the active transaction bound to the scope for each file input/output stream operation.
```

Comment:
```suggestion
On this page, we get you up and running with Sentry's File I/O Integration, so that it will automatically start a span out of the active transaction, bound to the scope for each file input/output stream operation.
```